ini/native.ini

```diff
@@ -113,7 +113,7 @@ build_flags = ${simulator_linux.build_flags} ${simulator_linux.release_flags}
 [simulator_macos]
 build_unflags     = -lGL -fstack-protector-strong
 build_flags       =
-  -DHAS_LIBBSD
+  -DHAS_LIBBSD -DGLM_ENABLE_EXPERIMENTAL
```

Comment:
Is this flag known also known be needed for macOS? I know we just added it for Windows. Maybe some Linux need it too…?

Reply:
On current `bugfix-2.1.x` / without this PR, both `simulator_macos_debug` and `simulator_macos_release` environments build the [default Simulator config](https://github.com/MarlinFirmware/Configurations/tree/bugfix-2.1.x/config/examples/Simulator) fine on my Intel-based Mac.

Reply:
Not required on Ubuntu 22.04.4 LTS

Reply:
Sometimes macOS needs it, and _probably_ it's ok to use the flag on Linux, or does it complain?